week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,14 +1,17 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// This function creates a random number between 0 and 1 and can contain 0 then return it's multiply by 10.
 function getNumber() {
     return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+// This function gets two parameters and returns concatenate second parameter to first parameter.  
 function s(w1, w2) {
     return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
+    return `${firstWord} ${secondWord} ${thirdWord}`;
```

Comment:
This is great!

What if I wanted to make this concat with `, `? Is there another method that would make it easier to maintain this function?